priv/repo/migrations/20230223162441_add_partners.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Logflare.Repo.Migrations.AddPartners do
+  use Ecto.Migration
+
+  def change do
+    create table :partners do
+      add :name, :binary
+      add :token, :binary
+      add :auth_token, :binary
```

Comment:
i feel that we should use the existing access tokens instead of using a field to store the token. this limits the number of tokens that a partner can use, makes rolling api keys harder.

Reply:
can add a "partner" scope to the access token.

Reply:
which entity has those access tokens? if you're mentioning the User model and make them "partners" I would disagree with that approach at least for now.

Would rather have this decoupled from the existing users to avoid more complexity there until we better understand what is fully needed. Plus, they are two completely different concepts: one uses Logflare, the other allows the usage of Logflare to third parties which is a new use case completely.

Reply:
I think all the concerns have been handled